src/components/EnemyStatsDisplay.js

```diff
@@ -1,57 +1,67 @@
-import {StyleSheet, Text, View, Image} from 'react-native';
+import {StyleSheet, Text, View, Image, FlatList} from 'react-native';
 import React from 'react';
+import atkIcon from '../assets/images/icon-sword.png';
+import defIcon from '../assets/images/icon-shield.png';
+import hpIcon from '../assets/images/icon-heart.png';
 
-const EnemyStatsDisplay = () => {
+const EnemyStatsDisplay = ({enemy}) => {
+  const addEquipmentStats = char => {
+    const stats = Object.assign({}, char);
+    stats.equipment.forEach(
+      equipment =>
+        (stats[equipment.affected_attribute] += equipment.affected_amount),
+    );
+    return stats;
+  };
+  const enemyInfo = addEquipmentStats(enemy);
+  console.warn(enemy);
+  // const enemyStats = [
+  //   {
+  //     img: atkIcon,
+  //     value: enemyInfo.atk,
+  //   },
+  //   {
+  //     img: defIcon,
+  //     value: enemyInfo.def,
+  //   },
+  //   {
+  //     img: hpIcon,
+  //     value: enemyInfo.hp,
+  //   },
+  //   {
+  //     img: atkIcon,
+  //     value: enemyInfo.agi,
+  //   },
+  // ];
+  console.log({enemy});
```

Comment:
I don't know if it was explained, but sometimes console.log on components root may be liars, or they are just annoying and print many times because of components rerender, to make sure it's working as expected, you can do this inside of a useEffect and track the effect for that propertie